site/src/pages/docs/server-docservice.mdx

```diff
@@ -211,18 +211,44 @@ files into the specific location in the class path.
 
   ```java
   import com.linecorp.armeria.server.annotation.Description;
+  import com.linecorp.armeria.server.annotation.ResponseObject;
 
-  @Description("A service that provides user information.")
+  @Description("""
```

Comment:
This is Java so we cannot use the String template. 😭

Reply:
Ah, I forgot this specification not merged yet 👀 (https://openjdk.org/jeps/8273943).
I'll fix it 😅